package/gluon-scheduled-domain-switch/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-scheduled-domain-switch
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-scheduled-domain-switch
+  TITLE:=Allows scheduled migrations between domains
+  DEPENDS:=+gluon-core +gluon-mesh-batman-adv
```

Comment:
I think the dependency on batman-adv can be dropped now?

Reply:
indeed it can now be dropped.